winton_kafka_streams/state/in_memory_key_value_store.py

```diff
@@ -0,0 +1,18 @@
+class InMemoryKeyValueStore:
```

Comment:
We already have some store implementation but it's a bit different from what's in upstream Kafka Streams now. Will revisit at a later point.